Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apk update && \
 RUN curl -SL ${KUBECTL_URI} -o kubectl && chmod +x kubectl
 RUN echo "${KUBECTL_SHA256}  kubectl" | sha256sum -c - || exit 10
 
-RUN pip install ijson
+RUN pip install ijson awscli
```

Comment:
Ideally this would be in a separate docker image, but I thought it might be easier to maintain as a single-branch repository. The cli is pretty small.